lib/ansible/plugins/filter/to_json.yml

```diff
@@ -5,56 +5,56 @@ DOCUMENTATION:
   short_description: Convert variable to JSON string
   description:
     - Converts an Ansible variable into a JSON string representation.
-    - This filter functions as a wrapper to the Python ``json.dumps()`` function.
+    - This filter functions as a wrapper to the Python C(json.dumps) function.
```

Comment:
I'm just thinking of this now . . . don't we have the ability to link to the Python docs? We could consider making these Python references links . . .

Reply:
I *think* we have, but I didn't want to go searching for the exact reference name... sorry :)